module_v2.py

```diff
@@ -0,0 +1,26 @@
+import git_commit_v2
```

Comment:
What is the purpose of this file?
The Data Frame suppose to be your easy assistant for creating the excel and should be used your creator script.

Give the files proper names that related to their content.

Reply:
it's a main file that takes a commit 
data frame just to print the content in the terminal 
I don't need data frame to write in excel file